### ISSUE TITLE
query string with wouter

### ISSUE THREAD

**Issue by songwang** [molefrog/wouter#261]
I don't see an example on how to declare route for query strings like   /search?keyword=wouter,
all the examples using the /search/:keyword type of URL,

can anybody give an example of the query string type of URL using wouter?

**Comment by penghuili** [comment]
seems wouter doesn't support this, In the end i just implemented a helper function, something like this 
```
function getQueryParams() {
  const obj = {};

  const searchParams = new URLSearchParams(window.location.search);
  searchParams.forEach((value, key) => {
    obj[key] = value;
  });

  return obj;
}
```

**Comment by molefrog** [comment]
We recently released v2.10.0 in which @HansBrende added some convenient methods for working with search strings. It isn't that advanced, however there is a helper method that allows you to build your own location hooks.

We put together an example in the release notes:
https://github.com/molefrog/wouter/releases/tag/v2.10.0

**Comment by molefrog** [comment]
Hey, the [v3.0.0-rc.1 is out](https://github.com/molefrog/wouter/releases/tag/v3.0.0-rc.1). We have added a native support for subscribing to current search string, that is performant and works well in SSR.
